cppcache/integration-test/testThinClientSecurityAuthentication.cpp

```diff
@@ -138,7 +140,7 @@ DUNIT_TASK_DEFINITION(LOCATORSERVER, CreateServer1)
         printf("Input to server cmd is -->  %s",
                cmdServerAuthenticator.c_str());
         CacheHelper::initServer(
-            1, nullptr, locHostPort,
+            1, "cacheserver_notify_subscription.xml", locHostPort,
             const_cast<char *>(cmdServerAuthenticator.c_str()));
```

Comment:
This should be fixed too. Really bad to be taking stripping the `const` off here. The funny thing is the method takes a `const`. The method should really be change to take a `const std::string&`. I know this might feel out of scope but it's right there!

Reply:
N/A. Test deleted.